docs/README.md

```diff
@@ -12,6 +12,7 @@ Create a file named `config.yaml` based off `config.yaml.example` with the follo
 - **job_search_fields**: Filter of Jenkins Jobs to included in report, e.g. DFG-ceph-rhos. To search for multiple fields, seperate them by comma, e.g. DFG-ceph-rhos,DFG-all-unified. Allows for regex searches as well, e.g. ^DFG-ceph,rgw$
 - **filter_param_name**: Optional field that instructs Jeeves to skip any build that lacks the corresponding value of the given build parameter. Must be used on in conjunction with **filter_param_value**
 - **filter_param_value**: Optional field that instructs Jeeves to skip any build that lacks this value for the corresponding build parameter name. Must be used in conjunction with **filter_param_name**
+- **stage_log**: Optional dict field that instructs Jeeves how to build an url to a log file for a corresponding failed build stage
```

Comment:
Can this data not be gotten from the Jenkins API directly?

Reply:
No, it can't be gotten from the Jenkins API directly. There is no direct API mapping between logs and stage names. However there are many common stages in infrared for all DFGs. We can try to create a basic mapping logs - stage name and put in config.example, which user can extend with DFG specific stages. Does it sound like a way to go?

Reply:
One thing I'm doing downstream that may be useful here is I run a tool that maps the stage to its junit xml.  I then parse the stage and failure and search through all other jobs in other DFGs to identify if the failure exists in other jobs as well.  I might be useful to have some type of extra report show cases the failures that impact you, the stage they happened, and all other DFGs that have had the same failure happen to them.  That way you have a better idea if this issue is limited to your test run or if its impacting a large number of DFGs.

Reply:
Sounds great. I would say that should be a separate tool, though, and we can just link from the jeeves report to it.

Reply:
Could also be a separate run mode, extension could be done via flag created in #192

Reply:
@nathan-weinberg I am all for separate run mode for detecting same failures in multiple DFGs jobs, however I think I can use @jamepark4  mapping function in my code instead hard-coding stage to log files. Do you have code already you can share? @jamepark4

Reply:
@ciecierski apologies I was trying to highlight the parsing and grouping a failures between all DFGs with my comment. The mapping is done in the same way you describe it, i.e. dictionary map with search patterns to relevant stages.  I can take a look to see if there are ways to dynamically map stage names to a corresponding log path, but when I wrote the code I did not see any great methods to do that.